snapshots/.storage-layout-aave-v2

```diff
@@ -36,5 +36,7 @@
 |---------------+-------------------------------------------------+------+--------+-------+-----------------------------------------|
 | poolToken     | address                                         | 251  | 0      | 20    | src/aave-v2/SupplyVault.sol:SupplyVault |
 |---------------+-------------------------------------------------+------+--------+-------+-----------------------------------------|
-| __gap         | uint256[47]                                     | 252  | 0      | 1504  | src/aave-v2/SupplyVault.sol:SupplyVault |
+| recipient     | address                                         | 252  | 0      | 20    | src/aave-v2/SupplyVault.sol:SupplyVault |
+|---------------+-------------------------------------------------+------+--------+-------+-----------------------------------------|
+| __gap         | uint256[48]                                     | 253  | 0      | 1536  | src/aave-v2/SupplyVault.sol:SupplyVault |
```

Comment:
Why it wasn't breaking the CI ?

Reply:
>Note that my storage checker is not failing because we're extending the storage of the supply vault using a storage word previously covered by an OZ _gap!

Ok my bad, we are using only your smart storage check here right ?

Reply:
I also updated the snapshot

Reply:
Yes but I'm wondering why it was not breaking before this Pr

Reply:
Ah it is breaking if you look at @pakim249CAL's PR